pages/1.12/gui/index.md

```diff
@@ -6,8 +6,44 @@ menuWeight: 40
 excerpt: Using the web interface for DC/OS management
 ---
 
-The DC/OS web interface provides a rich graphical view of your DC/OS cluster.
+The DC/OS web interface provides a rich graphical view of your DC/OS cluster. With the web interface you can view the current state of your entire cluster and DC/OS services. The web interface is installed as a part of your DC/OS installation.
 
-With the web interface you can view the current state of your entire cluster and DC/OS services. The web interface is installed as a part of your DC/OS installation.
+![Dashboard](/1.12/img/dashboard-ee-1-12.png)
 
-Additionally, there is a User Menu on the upper-left side of the web interface that includes links for [cluster overview](/1.11/gui/cluster), [linked clusters](/1.11/administering-clusters/multiple-clusters/cluster-links), documentation, CLI installation, and user sign out.
+*Figure 1 - Dashboard*
+
+# Web Interface Navigation Menu
+On the left hand sidebar, you will see the web interface navigation menu:
+- [Dashboard](/1.12/gui/dashboard/) (default view)
+- [Services](/1.12/gui/services/)
+- [Jobs](/1.12/gui/jobs/)
+- [Catalog](/1.12/gui/catalog/)
+- [Nodes](/1.12/gui/nodes/)
+- [Networking](/1.12/gui/networking/)
+- [enterprise][Secrets](/1.12/gui/secrets/) (available only in DC/OS Enterprise).[/enterprise]
+- [Cluster](/1.12/gui/cluster/)
+- [Components](/1.12/gui/components/)
+- [Settings](/1.12/gui/settings/)
+- [Organization](/1.12/gui/organization/)
+
```

Comment:
The navMenu on the left already lists these items, so this list is redundant. Moreover, if we re-arrange the list at the left, we will have to edit this page. I suggest that this list be deleted.